build

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+mvn install
```

Comment:
Und gibt es vielleicht auch ein Kommando, bei dem das Ganze nicht auch noch gleichzeitig in einen Folder im Home installiert wird?